test/parallel/test-cluster-worker-constructor.js

```diff
@@ -8,7 +8,7 @@ var cluster = require('cluster');
 var worker;
 
 worker = new cluster.Worker();
-assert.equal(worker.suicide, undefined);
+assert.equal(worker.exitedVoluntarily, undefined);
```

Comment:
These tests should probably only be changed when / if `.suicide` is removed, that way we retain test coverage for it. One or two can be changed but not all of them.